node/pkg/db/governor.go

```diff
@@ -22,29 +22,81 @@ type GovernorDB interface {
 	DeleteTransfer(t *Transfer) error
 	DeletePendingMsg(k *PendingTransfer) error
 	GetChainGovernorData(logger *zap.Logger) (transfers []*Transfer, pending []*PendingTransfer, err error)
+	StoreSignedVAA(v *vaa.VAA) error
+	GetSignedVAABytes(id VAAID) (b []byte, err error)
 }
 
 type MockGovernorDB struct {
+	storedVAAs       map[VAAID][]byte
+	transfers        map[string]*Transfer
+	pendingTransfers map[string]*PendingTransfer
 }
 
 func (d *MockGovernorDB) StoreTransfer(t *Transfer) error {
+	if d.transfers == nil {
+		d.transfers = make(map[string]*Transfer)
+	}
+	d.transfers[t.MsgID] = t
 	return nil
 }
 
 func (d *MockGovernorDB) StorePendingMsg(k *PendingTransfer) error {
+	if d.pendingTransfers == nil {
+		d.pendingTransfers = make(map[string]*PendingTransfer)
+	}
+	d.pendingTransfers[k.Msg.MessageIDString()] = k
 	return nil
 }
 
 func (d *MockGovernorDB) DeleteTransfer(t *Transfer) error {
+	if d.transfers != nil {
+		delete(d.transfers, t.MsgID)
+	}
 	return nil
 }
 
 func (d *MockGovernorDB) DeletePendingMsg(pending *PendingTransfer) error {
+	if d.pendingTransfers != nil {
+		delete(d.pendingTransfers, pending.Msg.MessageIDString())
+	}
 	return nil
 }
 
 func (d *MockGovernorDB) GetChainGovernorData(logger *zap.Logger) (transfers []*Transfer, pending []*PendingTransfer, err error) {
-	return nil, nil, nil
+	if d.transfers != nil {
+		for _, t := range d.transfers {
+			transfers = append(transfers, t)
```

Comment:
Could this ever be too big and blow up the stack?  Or is transfers a pointer to an array?

Reply:
This mock is only used in testing, where we only ever insert a couple, so that is not an issue.